packages/fields/src/types.ts

```diff
@@ -1,5 +1,5 @@
 type PostStatus =
-	| 'published'
+	| 'publish'
```

Comment:
This is also fixed in `trunk` as well as part of https://github.com/WordPress/gutenberg/commit/f992ba752021befdee058f68500607a73fa94673 😅